packages/plugin-ext/src/common/plugin-api-rpc.ts

```diff
@@ -443,6 +450,13 @@ export interface OpenDialogOptionsMain {
  * Options to configure the behaviour of a file save dialog.
  */
 export interface SaveDialogOptionsMain {
+
+    /**
+     * Dialog title.
+     * This parameter might be ignored, as not all operating systems display a title on save dialogs (for example, macOS).
```

Comment:
> not all operating systems display a title on save dialogs (for example, macOS)

[minor]: Maybe macOS is not a good example here. I can see the _Custom Open Dialod_ title. I am on `11.2.2 (20D80)` if this matters.

![Screen Shot 2021-03-23 at 17 45 20](https://user-images.githubusercontent.com/1405703/112184489-8ccb7f00-8bff-11eb-9055-295c8ff6c96a.jpg)